frontend/src/metabase/qb/components/modes/NativeMode.jsx

```diff
@@ -1,10 +1,11 @@
 /* @flow */
 
 import type { QueryMode } from "metabase/meta/types/Visualization";
+import CompoundQueryAction from "../actions/CompoundQueryAction";
 
 const NativeMode: QueryMode = {
     name: "native",
-    actions: [],
+    actions: [CompoundQueryAction],
```

Comment:
MBQL queries already have a lot of actions. I'll probably add it in on a re-organization at some point